README.md

```diff
@@ -1285,6 +1286,24 @@ Then install the new AWX Operator by following the instructions in [Basic Instal
 
 Once the new AWX Operator is up and running, your AWX deployment will also be upgraded.
 
+#### Disable IPV6 for install and upgrade
+Starting with  AWX Operator release 0.24.0 ipv6 had been enabled in ngnix configuration which caused upgrades/new install's to fail.
```

Comment:
@rakesh561 I revised this a bit, what do you think?

```
Starting with  AWX Operator release 0.24.0, [IPv6 was enabled in ngnix configuration](https://github.com/ansible/awx-operator/pull/950) which causes upgrades and installs to fail in environments where IPv6 is not allowed.  Starting in  1.1.1 release, you can set the `ipv6_disabled` flag on the AWX spec.  If you need to use an AWX Operator version between 0.24.0 and 1.1.1 in an IPv6 disabled environment, it is suggested to enable ipv6 on worker nodes.

In order to disable ipv6 on ngnix configuration (awx-web container), add following to the AWX spec.
```

Reply:
@rooftopcellist  Updated the requested changes it makes sense.Only thing is unable to make disable ipv6 section highlighted like other sections.